pinata-ssh-forward.sh

```diff
@@ -1,4 +1,14 @@
 #!/usr/bin/env bash
+
+ssh-add -L > /dev/null
+if [ ! $? -eq 0 ]
+then
+    RED_C='\033[0;31m'
+    RESET_C='\033[0m'
+    echo -e "${RED_C}SSH Agent doesn't have any identities, consider running:${RESET_C}"
+    echo "    ssh-add"
+fi
+
 set -eo pipefail
```

Comment:
we move `set -e` down since it would catch the failing `ssh-add` above which would prevent us from showing the hint to the user